client/utils/useDefaultFields.js

```diff
@@ -61,14 +63,15 @@ export const useConnectionsField = (isEditField, fields, connections, onConnecti
     return _.remove(fields, { property: 'connection' });
   }
 
+  const isConnectionListingLimitExceeded =  connections.length >= CONNECTIONS_LIST_LIMIT;
   const defaults = {
     property: 'connection',
     label: 'Connection',
```

Comment:
```suggestion
    label: isConnectionListingLimitExceeded ? 'Connection Name' : 'Connection',
```

I think when it's free text, we should specify that we need the connection name, rather than ID which I initially assumed it would be
![image](https://github.com/auth0-extensions/auth0-delegated-administration-extension/assets/2720466/69825c9f-0bda-4121-bece-a92e11a0ea03)